src/main/java/FuncTest/TestFunc/Function.java

```diff
@@ -35,6 +37,9 @@ public HttpResponseMessage run(
 			return getHourByWeek(queryResult, s, c);
 		case globals.HOMEWORK_GET_UPCOMING_INTENT_NAME:
 			return getUpcomingHomework(queryResult, s, c);
+		case globals.TEST_INTENT:
```

Comment:
Please remove this from this\ re-define the intent.

Reply:
@ShaiZeevi what should I do with it?
You added it when we tried integrating Tasks API with the bot

Reply:
@yoavz1997 just remove the usage of it in the code